kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -31,25 +31,53 @@
 import org.kitesdk.data.spi.Registration;
 
 /**
- * <p>Convenience methods for working with {@link Dataset} instances.</p>
- *
+ * Methods for working with {@link Dataset} instances.
+ * <p>
+ * <b>URIs</b><br /><br />
+ * All methods require a URI that identifies a dataset, view, or
+ * repository. The URI must begin with the scheme {@code dataset:}, 
+ * {@code view:}, or {@code repo:}. The remainder of the URI is 
+ * implementation specific, depending on the dataset scheme.
+ * <p>
+ * For example, the URI {@code dataset:hive:movies/ratings} 
+ * references a Hive dataset named <i>ratings</i> in the
+ * <i>movies</i> namespace.
+ * <p>
+ * The URI {@code view:hdfs:/user/me/movies/ratings?year=2015&month=3}
+ * references a view of an HDFS dataset named <i>ratings</i> in the
+ * <i>movies</i> namespace in the {@code /user/me} path. The view 
+ * is filtered to include records from only March, 2015.
+ * <p>
+ * See <a href="http://kitesdk.org/docs/current/URIs.html">Dataset and View
+ * URIs</a>.
+ * <p>
+ * <b>Entities</b><br /><br />
+ * <i>Entities</i> are analagous to <i>records</i> in database terminology.
+ * The term is used in the API to emphasize that an entity can include not
+ * only primitive objects, but also complex objects such as hash maps. 
+ * 
  * @since 0.8.0
  */
 public class Datasets {
 
   /**
-   * Load a {@link Dataset} or {@link View} for the given {@link URI}.
+   * Load a {@link Dataset} or {@link View} for the given {@link URI} and
+   * retrieve its metadata.
```

Comment:
What does load mean if not to retrieve the dataset's metadata? I don't think the addition is needed.